index.js

```diff
@@ -16,6 +17,18 @@ function lint(textEditor) {
 
 	const dir = pkgDir.sync(path.dirname(filePath));
 
+	// no package.json
+	if (!dir) {
+		return [];
+	}
+
+	// check if package.json has 'xo'
+	const rgx = new RegExp('xo', 'g');
+	const pkg = fs.readFileSync(path.join(dir, 'package.json'), 'utf8');
```

Comment:
Mostly just a preference, but I find it cleaner to set the `cwd` than joining paths, whenever there's a `cwd` option available.